### DISCUSSION TITLE
4 decimal display in DRO

### DISCUSSION THREAD

**Discussion by impartridge** [winder/Universal-G-Code-Sender#2193] [Help]
Is it possible to change the DRO to display 4 decimal point precision? I've played around with truncation in options, but still only displays 3 decimal points

**Comment by breiler** [comment]
The truncation will only apply to the gcode that is being sent, not what is received from the controller.

The numbers of decimals in the DRO are defined here:
https://github.com/winder/Universal-G-Code-Sender/blob/master/ugs-platform/ugs-platform-plugin-dro/src/main/java/com/willwinder/ugs/nbp/dro/panels/CoordinateLabel.java#L12

However, at least on my controller, the coordinate will only be reported from GRBL with 3 decimals. If I try to go to 0.0001 nothing happens and GRBL reports 0.000 back from the status report.
